frontend/lib/identification/qr_code_scanner/qr_code_scanner.dart

```diff
@@ -20,12 +22,29 @@ class QrCodeScanner extends StatefulWidget {
 }
 
 class _QRViewState extends State<QrCodeScanner> {
+  bool _hasCameraIssues = false;
+
+  @override
+  void initState() {
+    super.initState();
+    // Workaround for https://github.com/juliansteenbakker/mobile_scanner/issues/698
+    // Check once the qr code scanner was initialized if the device has camera issues
+    // Depending on that set a controller with predefined camera solution to fix that qr code reading issues
+    WidgetsBinding.instance.addPostFrameCallback((_) async {
+      setState(() async {
+        _hasCameraIssues = await isDeviceWithCameraIssues();
+      });
+    });
+  }
+
   final MobileScannerController _controller = MobileScannerController(
-    torchEnabled: false,
-    detectionSpeed: DetectionSpeed.normal,
-    formats: [BarcodeFormat.qrCode],
-    returnImage: false,
-  );
+      torchEnabled: false, detectionSpeed: DetectionSpeed.normal, formats: [BarcodeFormat.qrCode], returnImage: false);
+  final MobileScannerController _controllerPredefinedCameraResolution = MobileScannerController(
+      torchEnabled: false,
+      detectionSpeed: DetectionSpeed.normal,
+      formats: [BarcodeFormat.qrCode],
+      returnImage: false,
+      cameraResolution: const Size(640, 480));
```

Comment:
I am not entirely sure if it works, but you could make the _controller non-final and simply overwrite if/once we detect it is a problematic device (instead of setting the `_hasCameraIssues` flag).

Reply:
That does not work